gulpfile.js

```diff
@@ -43,16 +43,16 @@ var testFiles = [
   // Disable tests which need to be rewritten based on changes introduced by
   // the following changes: https://github.com/chartjs/Chart.js/pull/2346
   '!./test/core.layoutService.tests.js',
-  '!./test/defaultConfig.tests.js',
+  '!./test/defaultConfig.tests.js'
 ];
 
 gulp.task('bower', bowerTask);
 gulp.task('build', buildTask);
 gulp.task('package', packageTask);
 gulp.task('coverage', coverageTask);
 gulp.task('watch', watchTask);
-gulp.task('jshint', jshintTask);
-gulp.task('test', ['jshint', 'validHTML', 'unittest']);
+gulp.task('lint', lintTask);
```

Comment:
I thought we were also changing the test tasks and `gulp unittestWatch` was going way in favour of `gulp test -watch`. Or will that be a separate PR?